pkg/controller/controller.go

```diff
@@ -441,7 +441,7 @@ func (c *Controller) createManagedClusterResourcesForBinding(mbPair *types.Model
 	}
 
 	// Create Namespaces
-	err = managed.CreateNamespaces(mbPair, c.managedClusterConnections)
+	err = managed.CreateNamespaces(mbPair, filteredConnections)
```

Comment:
Why this change to use filteredConnections?

Reply:
CreateNamespaces used to use a util method to get the filtered connections from the available.  This passes the filtered connections in instead.  The available connections are not needed otherwise.

Reply:
To be clear, this pushes the util method up and out of CreateNamespaces so that the mock connections can be passed in from the test.